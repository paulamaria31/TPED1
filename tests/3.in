30
OVXVOOXXX
OXOXOOVOV
XXVOVVOXV
VOVOOXXOV
OOXVXOXXX
XOOVOOXOV
VVVXVXOOV
VXOOXXXOO
XVVVOXVOX
OVVVXOXOX
OOVXVVXXX
OOXVOVVOX
OOVOXVXVO
XOOVVXVXO
XVVVXOOXV
XVOXOVOOV
OVXXXOVVV
OXOVOXOVO
VXVVVVXVV
XOVOOXXOX
OOXOOXVVO
VXXXXVVXV
XOXOXVXVX
OVXOOOXOV
XXVOOXOOV
OXOOVVOXV
XXVVOOVVO
OOVVOOOOX
XXXXVVVOO
XOOOVOVOX
