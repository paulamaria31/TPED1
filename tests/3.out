Tabuleiro 1 com vitoria [X]
Tabuleiro 2 invalido
Tabuleiro 3 em andamento [O: sem jogada mestre]
Tabuleiro 4 invalido
Tabuleiro 5 invalido
Tabuleiro 6 invalido
Tabuleiro 7 em andamento [proximo jogador indefinido]
Tabuleiro 8 em andamento [proximo jogador indefinido]
Tabuleiro 9 em andamento [O: (1,2)]
Tabuleiro 10 em andamento [proximo jogador indefinido]
Tabuleiro 11 invalido
Tabuleiro 12 invalido
Tabuleiro 13 invalido
Tabuleiro 14 em andamento [proximo jogador indefinido]
Tabuleiro 15 em andamento [O: sem jogada mestre]
Tabuleiro 16 invalido
Tabuleiro 17 em andamento [O: sem jogada mestre]
Tabuleiro 18 invalido
Tabuleiro 19 invalido
Tabuleiro 20 com vitoria [O]
Tabuleiro 21 invalido
Tabuleiro 22 invalido
Tabuleiro 23 invalido
Tabuleiro 24 invalido
Tabuleiro 25 em andamento [X: (1,3)]
Tabuleiro 26 invalido
Tabuleiro 27 em andamento [X: (1,3)]
Tabuleiro 28 invalido
Tabuleiro 29 invalido
Tabuleiro 30 invalido
